spec/Appendix C -- Built-in Definitions.md

```diff
@@ -0,0 +1,366 @@
+expected as an input type, any string (such as `"4"`) or integer (such as `4`) input value will be accepted as
+an ID.
+"""
+scalar ID
+```
+
+## Directives
+
+```graphql
+"""
+Directs the executor to include this field or fragment only when the `if` argument is true
+"""
+directive @include(
+  """
+  Included when true.
+  """
+  if: Boolean!
+) on FIELD | FRAGMENT_SPREAD | INLINE_FRAGMENT
+
+"""
+Directs the executor to skip this field or fragment when the `if` argument is true.
+"""
+directive @skip(
+  """
+  Skipped when true.
+  """
+  if: Boolean!
+) on FIELD | FRAGMENT_SPREAD | INLINE_FRAGMENT
+
+"""
+Marks the field, argument, input field or enum value as deprecated
+"""
+directive @deprecated(
+  """
+  The reason for the deprecation
+  """
+  reason: String = "No longer supported"
+) on FIELD_DEFINITION | ARGUMENT_DEFINITION | ENUM_VALUE | INPUT_FIELD_DEFINITION
+
+"""
+Exposes a URL that specifies the behavior of this scalar.
+"""
+directive @specifiedBy(
+  """
+  The URL that specifies the behavior of this scalar.
+  """
+  url: String!
+) on SCALAR
+```
+
+## Introspection types
+
+```graphql
+"""
+A GraphQL Schema defines the capabilities of a GraphQL server. It exposes all available types and directives
+on the server, as well as the entry points for query, mutation, and subscription operations.
+"""
+type __Schema {
+  description: String
+
+  """
+  A list of all types supported by this server.
+  """
+  types: [__Type!]!
+
+  """
+  The type that query operations will be rooted at.
+  """
+  queryType: __Type!
+
+  """
+  If this server supports mutation, the type that mutation operations will be rooted at.
+  """
+  mutationType: __Type
+
+  """
+  If this server support subscription, the type that subscription operations will be rooted at.
+  """
+  subscriptionType: __Type
+
+  """
+  A list of all directives supported by this server.
+  """
+  directives: [__Directive!]!
+}
+
+"""
+The fundamental unit of any GraphQL Schema is the type. There are many kinds of types in GraphQL as
+represented by the `__TypeKind` enum.
+
+Depending on the kind of a type, certain fields describe information about that type.
+"""
+type __Type {
+  kind: __TypeKind!
+  name: String
+  description: String
+  """
+  must be non-null for OBJECT and INTERFACE, otherwise null.
+  """
+  fields(includeDeprecated: Boolean = false): [__Field!]
+  """
+  must be non-null for OBJECT and INTERFACE, otherwise null.
+  """
+  interfaces: [__Type!]
+  """
+  must be non-null for INTERFACE and UNION, otherwise null.
+  """
+  possibleTypes: [__Type!]
+  """
+  must be non-null for ENUM, otherwise null.
+  """
+  enumValues(includeDeprecated: Boolean = false): [__EnumValue!]
+  """
+  must be non-null for INPUT_OBJECT, otherwise null.
+  """
+  inputFields(includeDeprecated: Boolean = false): [__InputValue!]
+  """
+  must be non-null for NON_NULL and LIST, otherwise null.
+  """
+  ofType: __Type
+  """
+  may be non-null for custom SCALAR, otherwise null.
```

Comment:
```suggestion
  Must be non-null for OBJECT and INTERFACE, otherwise null.
  """
  fields(includeDeprecated: Boolean = false): [__Field!]
  """
  Must be non-null for OBJECT and INTERFACE, otherwise null.
  """
  interfaces: [__Type!]
  """
  Must be non-null for INTERFACE and UNION, otherwise null.
  """
  possibleTypes: [__Type!]
  """
  Must be non-null for ENUM, otherwise null.
  """
  enumValues(includeDeprecated: Boolean = false): [__EnumValue!]
  """
  Must be non-null for INPUT_OBJECT, otherwise null.
  """
  inputFields(includeDeprecated: Boolean = false): [__InputValue!]
  """
  Must be non-null for NON_NULL and LIST, otherwise null.
  """
  ofType: __Type
  """
  May be non-null for custom SCALAR, otherwise null.
```